src/lib/grad.jl

```diff
@@ -124,15 +124,15 @@ julia> jacobian((a,d) -> prod(a, dims=d), [1 2; 3 4; 5 6], 2)
 !!! warning
     For arguments of any type except `Number` & `AbstractArray`, the result is `nothing`.
 
-```jldoctest; setup=:(using Zygote)
+```
```

Comment:
why you remove the doctest?

Reply:
Because one of the gradient calls now returns a FillArray, which was making tests fail. And because I the printing of that may change soon, I realised I'd edited the text to the new printing not the old... still maybe I should put it back?

Reply:
maybe the Fill is more confusing than helpful in this docstring, maybe we can keep the old version but remove the doctest. Or have the Fill but keep the doctest

Reply:
With this PR + tagged FillArrays, it looks like this, which I think is a bit ugly, and not so helpful:
```
julia> jacobian((a,t) -> sum(a .* t[1]) + t[2], [1,2,3], (4,5))
([4 4 4], nothing)

julia> gradient((a,t) -> sum(a .* t[1]) + t[2], [1,2,3], (4,5))  # gradient undersands the tuple
(3-element Fill{Int64}: entries equal to 4, (6, 1))
```
Maybe best is to restore the old printing with no doctest here:
```
julia> jacobian((a,t) -> sum(a .* t[1]) + t[2], [1,2,3], (4,5))
([4 4 4], nothing)

julia> gradient((a,t) -> sum(a .* t[1]) + t[2], [1,2,3], (4,5))  # gradient undersands the tuple
([4, 4, 4], (6, 1))
```

Reply:
agree

Reply:
Done.

(The new method below still returns a Fill in this case, btw.)